test/cpp/tensorexpr/test_kernel.cpp

```diff
@@ -36,7 +37,17 @@ void testKernel_1() {
   TensorExprKernel k(graph);
   std::vector<at::Tensor> inputs = {a, b};
   Stmt* s = k.getCodeGenStmt();
-  // TODO: verify stmt
+
+  std::ostringstream oss;
+  oss << *s;
+
+  // Check the IR we produced
+  const std::string& verification_pattern =
+      R"IR(
+# CHECK: for (int v = 0; v < 5; v++)
```

Comment:
Do we want to be that specific? Maybe just check that we fused two ops into a single loop nest?
Like:
```
# CHECK: for
# CHECK-NEXT: for
# CHECK-NOT: for
```

Reply:
As I've said in the comment below, these formulas have value. I'd rather keep them for now and revisit if it proves to be onerous to keep them consistent.

Reply:
Done.